src/transactions/feed/TransactionFeedV2.tsx

```diff
@@ -205,58 +219,15 @@ function mergeStandByTransactionsInRange({
   })
   const deduplicatedTransactions = deduplicateTransactions([...transactions, ...standByInRange])
   const sortedTransactions = sortTransactions(deduplicatedTransactions)
-  const transactionsFromAllowedNetworks = sortedTransactions.filter((tx) =>
-    allowedNetworks.includes(tx.networkId)
-  )
```

Comment:
Moved this filter to `sections` to ensure that if `allowedNetworks` change - `sections` will be re-processed.